vars/managerPipeline.groovy

```diff
@@ -177,8 +172,8 @@ def call(Map pipelineParams) {
                  description: (
                      'Extra environment variables to be set in the test environment, uses the java Properties File Format.\n' +
                      'Example:\n' +
-                     '\tSCT_STRESS_IMAGE.cassandra-stress=scylladb/cassandra-stress:3.13.0\n' +
-                     '\tSCT_N_DB_NODES=6\n' +
+                     '\tSCT_MGMT_RESTORE_EXTRA_PARAMS=--batch-size 50 --parallel 0\n' +
```

Comment:
There no need to update this one on every change, few examples are enough.

I would be too long to hold all the possibilities